src/internal/auth/interceptor.go

```diff
@@ -103,6 +104,20 @@ var authHandlers = map[string]authHandler{
 	"/identity.API/DeleteOIDCClient":        admin,
 	"/identity.API/DeleteAll":               admin,
 
+	//
+	// License API
+	//
+	"/license.API/Activate":          authDisabledOr(admin),
```

Comment:
> Just checking my reading of the code here: is the idea that an org admin is a cluster admin of the license server?

Correct, for now "cluster admin" in the server running the license service gives you access to the license service. There's not really a distinct notion of an "org admin", I wanted to wait for role bindings to add that.

> Would it ever make sense to run the license server with auth off, and if not, should we try to prevent users from doing that somehow (angry warnings or errors or something)?

For one there's a chicken and egg problem: you can't enable auth without enterprise, which requires a working license server. Also some customers use enterprise without an auth provider, so this case has to be supported. I don't think it's a special security concern compared to how our current enterprise server works.